upa/test/submissionIntervalTest.ts

```diff
@@ -928,6 +932,18 @@ describe("SubmissionInterval", () => {
       ).to.throw();
     });
 
+    it("dont attempt to merge submissions with different dupIdxs", () => {
+      // sA and sADup have the same proofs, and therefore the same submission
+      // Id.  They should NOT be considered for merging, and should not cause
+      // an error. (internal issue #1079).
```

Comment:
better not to refer to saturn issues in the open source repo

Reply:
Yeah, I wondered about this.  Hence the "internal".  I did want to be able to point at something to justify the test, so at least we have an internal record, otherwise it looks like kind of a stupid test.